omod/src/main/webapp/admin/maintenance/implementationIdForm.jsp

```diff
@@ -18,7 +18,7 @@
 				<td style="white-space: nowrap">
 				    <input type="text" name="${status.expression}" value="${status.value}" size="40" />
 				    <c:if test="${status.errorMessage != ''}">
-                        <span class="error">${status.errorMessage}</span>
+                        <span class="error">$<c:out value="${status.errorMessage}"/></span>
```

Comment:
Was the `$` character before the `<c:out` tag on this line left in on purpose?

Reply:
No, thanks for pointing that out